src/tools/vfs/vfs_server.c

```diff
@@ -211,7 +211,7 @@ static void vfs_server_mount(int idx, pid_t pid)
 
     fuse_fd = vfs_mount(pid);
     if (fuse_fd < 0) {
-        vfs_server_close_fd(idx);
+        vfs_server_remove_fd(idx);
```

Comment:
1. nice catch!
2. which test scenario triggered this issue? (pls add to PR desc)
3. shall we fix line 208 as well?